lib/webview.dart

```diff
@@ -19,9 +20,11 @@ class _ScreenerAppState extends State<ScreenerApp> {
   late final String _screenerHomeUrl =
       widget.debug ? "http://10.0.2.2:8000" : 'https://www.screener.in';
   late String paymentUrl = "'$_screenerHomeUrl/payment/capture/'";
```

Comment:
Are both double and single quotes needed in this string?

Reply:
Yes